src/openrct2-ui/audio/AudioMixer.cpp

```diff
@@ -470,8 +467,8 @@ namespace OpenRCT2::Audio
             float endvolume_f = static_cast<float>(endvolume) / SDL_MIX_MAXVOLUME;
             for (int32_t i = 0; i < length; i++)
             {
-                float t = static_cast<float>(i) / length;
-                data[i] = static_cast<int16_t>(data[i] * ((1 - t) * startvolume_f + t * endvolume_f));
+                float t = (float)i / length;
```

Comment:
```suggestion
                float t = static_cast<float>(i) / length;
```
Why would we revert back to c-style cast?